src/main/kotlin/no/nav/familie/ef/søknad/api/dto/søknadsdialog/EttersendingDto.kt

```diff
@@ -1,7 +1,6 @@
 package no.nav.familie.ef.søknad.api.dto.søknadsdialog
```

Comment:
Detter er sikkert noe vi har fra tidligere, men tenker at disse filene ikke bør ligge i `package no.nav.familie.ef.søknad.api.dto.søknadsdialog`. Vi kan heller lage en mappe for ettersending i dto-mappen. Da blir pakken `package no.nav.familie.ef.søknad.api.dto.ettersending`